calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
+    total += grades[i];
+  }
+
+  let avg = total / grades.length;
+
+  return avg;
```

Comment:
instead of storing the average as a variable, you could return the calculation return value directly
```suggestion
  return total / grades.length;
```